### ISSUE TITLE
prepare hurricanes Florence, Charlie, Andrew for storm replay

### ISSUE THREAD

**Issue by wwlwpd** [jasonfleming/asgs#446]
Boils down to:

# ensure historical data is in storm archive; if not find it and get it in there
# verify the nowification script works with them
# update the replay server configuration file to include the storms
# restart the replay server

**Comment by wwlwpd** [comment]
Florence has been added to the replay. Charley and Andrew are going
to require some on going work. Andrew, in particular, seems to be
only available as a best track file. It would be nice for the replay
stuff to have the ability to basically recreate work-alike "nowified"
best track and advisory forecast files based on the final best track -
I think it can be done; just need to work out the details.

**Comment by jasonfleming** [comment]
Yes, the Andrew hindcast (and any/all pre-2000s storms) will require us to run the simpler NWS=8 symmetric parametric vortex model b/c the latest GAHM vortex model requires shape information that the older files do not have. It would be neat to use storm_track_gen.pl to come up with forecast tracks (maybe semi-random) that do not match the eventual BEST track. Like you said, it is doable but will require an increment of effort.

**Comment by wwlwpd** [comment]
will revisit this when focus turns to replay server
